frontend/src/ui/player/Paella.tsx

```diff
@@ -465,6 +477,22 @@ const PAELLA_CONFIG = {
             order: 9,
             tabIndex: 9,
         },
+        "es.upv.paella.frameControlButtonPlugin": {
+            enabled: true,
+            side: "right",
+            order: 102,
+            tabIndex: 17,
```

Comment:
Those are some high numbers... is that intentional?

Reply:
Oh good call. I copied that from https://paellaplayer.upv.es/#/playground and forgot to adjust those numbers.

Reply:
I think the tabIndex: 17 is still there? Or is that intentional now?